OWNERS

```diff
@@ -34,7 +34,7 @@ filters:
   # go.{mod,sum} files relate to go dependencies, and should be reviewed by the
   # dep-approvers
   "go\\.(mod|sum)$":
-    required_reviewers:
-      - kubernetes/dep-approvers
+    approvers:
```

Comment:
So all this does is remove reviewers. The approve plugin doesn't support regex rn.

Reply:
review assignment inspects `approvers`.

Reply:
we already could not approve everything, and assigning a github team is an ineffectual pattern.

Reply:
aliases are also public, while github teams are opaque